corehq/apps/export/const.py

```diff
@@ -51,3 +51,4 @@
 
 FORM_EXPORT = 'form'
 CASE_EXPORT = 'case'
+MAX_EXPORTABLE_ROWS = 1000000
```

Comment:
https://github.com/dimagi/commcare-hq/pull/12438/commits/ebfda4b099950109625ac17f11bc59aed51ff561 moved it down by factor of 10